model/enduser/registry.yaml

```
@@ -0,0 +1,18 @@
+groups:
+  - id: registry.enduser
+    type: attribute_group
+    display_name: End User Attributes
+    brief: >
+      Describes the end user.
+    attributes:
+      - id: enduser.id
+        type: string
+        brief: "Unique identifier of an authenticated user in the system."
```

Do we have any recommendations on what this id should be ? login? email? anything user app wants to use?

Please adjust this note based on the answer

```suggestion
        brief: "Uniquely identifies end user in the system. It may be a login, email address, or another id used for authentication"
```